component/alertrules.jsonnet

```diff
@@ -0,0 +1,47 @@
+local kap = import 'lib/kapitan.libjsonnet';
+local kube = import 'lib/kube.libjsonnet';
+local inv = kap.inventory();
+local params = inv.parameters.system_upgrade_controller;
+
+local alert_rules =
+  kube._Object('monitoring.coreos.com/v1', 'PrometheusRule', 'system-upgrade-controller-alert-rules') {
+    metadata+: {
+      namespace: params.monitoring.prometheusRuleNamespace,
+      labels+: {
+        role: 'alert-rules',
+        prometheus: 'platform',
+      },
+    },
+    spec: {
+      groups: [
+        {
+          name: 'system-upgrade-controller.rules',
+          rules: [
+            {
+              alert: 'SYN_SystemUpgradeControllerMaintenanceHalted',
+              expr: 'kube_node_spec_unschedulable >= 1',
```

Comment:
This seems somewhat simplistic, this alert will fire if a node is marked unschedulable for any reason, not just because SUC can't drain it.